test/cli.py

```diff
@@ -722,7 +722,7 @@ def test_016_skip_compiled(self):
 
     def test_017_invalid_file_type(self):
         # Ensure that we get an error when running turnkey with invalid input_files
-        with self.assertRaises(exceptions.ArgError):
```

Comment:
This change to testing was needed because we changed from a bespoke error to the generic argparse error